.travis.yml

```diff
@@ -19,5 +19,6 @@ script:
   - set -e
   - pdd --file=/dev/null --exclude=qulice-maven-plugin/src/it/**/*.java
   - est --dir=est --file=/dev/null
+  - ./years.sh
```

Comment:
@dmzaytsev yes, it is :), adding `./` is required if given executable is not in the `PATH`